app/assets/stylesheets/core/_link.scss

```diff
@@ -1,6 +1,7 @@
 // TODO: support this modifier in govuk-frontend
```

Comment:
I'm guessing this didn't happen :-(.

Reply:
Nope, not yet. I have a note in my to do list.